pywinauto/hooks.py

```diff
@@ -275,14 +280,22 @@ def listen(self):
         atexit.register(windll.user32.UnhookWindowsHookEx, self.keyboard_id)
         atexit.register(windll.user32.UnhookWindowsHookEx, self.mouse_id)
 
-        message_pointer = MSG()
+        message = MSG()
 
         while self.mouse_is_hook or self.keyboard_is_hook:
-            msg = windll.user32.GetMessageW(message_pointer, 0, 0, 0)
-            windll.user32.TranslateMessage(byref(message_pointer))
-            windll.user32.DispatchMessageW(byref(message_pointer))
-            if message_pointer.message == self.WM_QUIT:
+            msg = windll.user32.GetMessageW(byref(message), 0, 0, 0)
+            if msg == -1:
+                print("Error. Sorry, I exit... :(")
+                hk.unhook_keyboard()
+                hk.unhook_mouse()
+                exit(0)
```

Comment:
I'm not sure `exit(0)` is what we need. Just break the loop, do not exit the whole Python process.